cypress/locators/simple-color-picker/locators.js

```diff
@@ -1,2 +1,4 @@
 // component preview locators
 export const SIMPLE_COLOR_PICKER_PREVIEW = 'div[data-component="simple-color-picker"] > ul';
+export const EXPERIMENTAL_SIMPLE_COLOR_PICKER = '[data-component="simple-color-picker"]';
+export const EXPERIMENTAL_SIMPLE_COLOR = 'div[data-component="simple-color"]';
```

Comment:
do we need div tag here? does the separate colour is in other tag then `div`
```suggestion
export const EXPERIMENTAL_SIMPLE_COLOR = '[data-component="simple-color"]';
```

Reply:
fixed: b81dcaf